### DISCUSSION TITLE
API is down in India location

### DISCUSSION THREAD

**Discussion by ThanikaiKing** [lukePeavey/quotable#194] [Report Downtime]
I am not able to access the API from the India location, but I am able to generate quotes when connecting to VPN.

**Comment by sidedwards** [comment]
Same with Europe

**Reply by lukePeavey** [reply]
I apologize for slow response!  The issues has been resolved

**Reply by ThanikaiKing** [reply]
I can see successful responses from the API from India. Thank you! @lukePeavey
